__tests__/app.test.js

```diff
@@ -143,4 +143,51 @@ describe.only('GET /api/articles', () => {
 
         })
     })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('should respond with 200 for a good request', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({ body }) => {
+            
+            expect(Array.isArray(body.comments)).toBe(true)
+            if (body.comments.length !== 0) {
+                body.comments.forEach((comment) => {
+                    if (Object.keys(comment).length !== 0) {
+                        expect(comment).toEqual(
+                            expect.objectContaining({
+                                comment_id: expect.any(Number),
+                                votes: expect.any(Number),
+                                created_at: expect.any(String), 
+                                author: expect.any(String), 
+                                body: expect.any(String), 
+                                article_id: expect.any(Number)
+                            })
+                        )
+
+                    }
+                   
+                })
+            }
+            expect(body.comments).toBeSorted({descending : true, key: 'created_at'})
+        })
+    })
+    test('should respond with 400 when passed an article id that is not uniform', () => {
+        return request(app)
+          .get('/api/articles/notAnID/comments')
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toBe('ID not exists');
+        });
+    })
+    test('status 404 responds with message if a valid input but no article', () => {
+        return request(app)
+        .get('/api/articles/9999/comments')
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe('Article doesn\'t exist')
+        })
```

Comment:
Great work handling the 2 errors. However, you are missing a happy oath test here. What if the article does exist but has no comments associated with it? For example article 13? This should respond with a 200 status and and empty array. Please do add this test